src/apis/Animated.bs.js

```diff
@@ -106,7 +106,9 @@ function reset(prim) {
 
 var StyleProp = {};
 
-var make = ReactNative.Animated.createAnimatedComponent(ReactNative.FlatList);
+var make = ReactNative.Animated.createAnimatedComponent(function (prim) {
+      return ReactNative.FlatList(prim);
```

Comment:
This change in output will most likely cause problems. I wonder why it occurs only for Flatlist?

Reply:
imho : we should import Animated.* instead of recreating them